src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -3213,36 +3213,44 @@ void Unit::_UpdateSpells(uint32 time)
 
 void Unit::_UpdateAutoRepeatSpell()
 {
-    // check "realtime" interrupts
-    if ((GetTypeId() == TYPEID_PLAYER && ToPlayer()->isMoving()) || IsNonMeleeSpellCast(false, false, true, m_currentSpells[CURRENT_AUTOREPEAT_SPELL]->m_spellInfo->Id == 75))
+    SpellInfo const* spellProto = nullptr;
+    if (m_currentSpells[CURRENT_AUTOREPEAT_SPELL])
+        spellProto = m_currentSpells[CURRENT_AUTOREPEAT_SPELL]->m_spellInfo;
+
+    if (!spellProto)
+        return;
```

Comment:
please use brackets on `if` blocks, even when it's 1-line

Reply:
That's scary looking but okay, if you guys prefer that code-style 👍

Reply:
Oh and which approach?

```
    if (!spellProto) {
        return;
    }
```
OR
```
    if (!spellProto)
    {
        return;
    }
```

Reply:
this project uses the second one :wink: